src/audio/sfx.c

```diff
@@ -318,7 +326,11 @@ void Audio_ChooseActiveSfx(u8 bankId) {
                 entry->dist = 0.0f;
             } else {
                 tempf1 = *entry->posY * 1;
+#ifdef OOT_DEBUG
                 entry->dist = (SQ(*entry->posX) + SQ(tempf1) + SQ(*entry->posZ)) * 1;
+#else
+                entry->dist = (SQ(*entry->posX) + SQ(tempf1) + SQ(*entry->posZ)) / 10.0f;
```

Comment:
is the compiler smart enough to turn `/ 1.0f` into `* 1.0f`? If so perhaps we could give a name to this thing which is 1.0f in retail but 10.0f in debug. I notice there's a similar thing in Audio_SetSfxProperties too (1.0f in retail but 10.0f in debug)

Reply:
Yeah, I was able to get it to work. That's probably why `* 1.0f` didn't work, it needed to be `/ 1.0f`. I made the macro and applied it in 2 places